src/VisualStudio/IntegrationTest/IntegrationTests/VisualStudioIntegrationTests.csproj

```diff
@@ -61,6 +61,7 @@
     <Reference Include="UIAutomationTypes" />
     <PackageReference Include="EnvDTE" Version="$(EnvDTEVersion)" />
     <PackageReference Include="EnvDTE80" Version="$(EnvDTE80Version)" />
+    <PackageReference Include="Microsoft.Test.Apex-vsuvscore" Version="$(MicrosoftTestApexVersion>)" />
```

Comment:
Since the package ID is going to change before this goes public, do you want to use a macro for the ID now, or just use search and replace once later? 
I suspect search and replace is better since the ID will only change once, but thought I'd offer.

Reply:
This is not going to pass CI due to the Include name not matching the version macro.

Reply:
If you mean ">" in the end, then fixed. Thank you! 

Now waiting for the NuGet package to be public available.  Then, will update the Nuget.Config. Do you see more issues with this?

---
In reply to: [162495805](https://github.com/dotnet/roslyn/pull/22201#discussion_r162495805) [](ancestors = 162495805)